tox.ini

```diff
@@ -24,44 +24,41 @@ addopts = --cov completion --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements
 
 [testenv]
-deps =
-    django18: Django>=1.8,<1.9
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
+deps = 
     django22: Django>=2.2,<2.3
+    django30: Django>=3.0,<3.1
     drf37: djangorestframework<3.8.0
     drf38: djangorestframework<3.9.0
-    drf39: djangorestframework<4.0.0
+    drf39: djangorestframework<3.10.0
```

Comment:
fixed the dependency to install the correct version, Django rest framework has 3.10 after 3.9 ([source](https://www.django-rest-framework.org/community/release-notes/#versioning))